vm/devices/vmbus/vmbus_server/src/lib.rs

```diff
@@ -887,6 +897,10 @@ impl ServerTask {
             VmbusRequest::Start => {
                 if !self.running {
                     self.running = true;
+                    if self.unstick_on_start {
+                        tracing::info!("lost synic bug fix is not in yet, call unstick_channels to mitigate the issue.");
+                        self.unstick_channels(false);
```

Comment:
Also set unstick_on_start to false after this, since we've already done the unstick.